tests/unit/core.js

```diff
@@ -424,8 +424,7 @@ exports.argsInCatchReused = function (test) {
 exports.testRawOnError = function (test) {
   JSHINT(';', { maxerr: 1 });
   test.equal(JSHINT.errors[0].raw, 'Unnecessary semicolon.');
-  test.equal(JSHINT.errors[1].raw, 'Too many errors.');
-  test.equal(JSHINT.errors[2], null);
+  test.equal(JSHINT.errors[1], null);
```

Comment:
This change may be sufficient to cause the test to pass, but it subvert the original intent. Can you add another test which verifies that when `maxErr` is set to `1`, JSHint reports "Too many errors" after encountering a single error?

Reply:
You seem right, I overlooked the original intent of this test. I believe replacing the semicolon warning with an actual error like "Unrecoverable syntax error" (if the input is `}`) would help the test retain it's original intent. 
Would replacing `;` with `}` here and adding back the test for "Too many errors." be sufficient?

Reply:
Let's do both

Reply:
Modified the test to reflect it's original indented behavior and added another test to accommodate for this PR change. Let me know if it looks good!